### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by Gomni** [ObscureGreenTea/ForceCloneFixed#6]
How do you use the mod? I'm new to this and i don't know how to use it

**Comment by TheKfcBandit** [comment]
click on person, yoink thy style, dont be a cunt tho

**Comment by Gomni** [comment]
Is there a option i have to turn on? I don't see force clone

**Comment by TheKfcBandit** [comment]
Nope, when you click on someone at the top of the quick menu you should see the red text. Make sure you are using melon loader v.0.3.0. make sure it is In your "mods" folder inside vrchats directory. If that doesn't fix it, when you boot up the game in the melon loader console look for text saying that "forceclonefixed" has been loaded. If it isn't in the console contact me or if there is no console reinstall "melon loader"

**Comment by Gomni** [comment]
Hmm. This is quite weird for me.
It keeps saying unable to read VR path registry
